texar/modules/classifiers/bert_classifiers.py

```diff
@@ -80,235 +91,172 @@ def __init__(self, hparams=None):
             else:
                 logit_kwargs = logit_kwargs.todict()
 
-            self.logits_layer = nn.Linear(
-                self._hparams.hidden_size, self._num_classes, **logit_kwargs)
-
-        else:
-            self.logits_layer = None
-
-        self.step_iteration = 0
+            if self._hparams.clas_strategy == 'all_time':
+                self._logits_layer = nn.Linear(
+                    self._hparams.hidden_size * self._hparams.max_seq_length,
+                    self.num_classes,
+                    **logit_kwargs)
+            else:
+                self._logits_layer = nn.Linear(self._hparams.hidden_size,
+                                               self.num_classes,
+                                               **logit_kwargs)
+        self.is_binary = None
 
     @staticmethod
     def default_hparams():
         r"""Returns a dictionary of hyperparameters with default values.
 
         .. code-block:: python
 
-            {
-                # (1) Different modules for sentence encoder
-                # (2) Additional hyperparameters
-                "num_classes": 2,
-                "logit_layer_kwargs": None,
-                "clas_strategy": "cls_time",
-                "max_seq_length": None,
-                "dropout": 0.1,
-                "name": "bert_classifier"
+        {
+            # (1) Same hyperparameters as in BertEncoder
+            ...
+            # (2) Additional hyperparameters
+            `num_classes`: 2,
```

Comment:
If this is a code example, we should use quotes (`"`) instead of backticks (`` ` ``).